docs/guides/auth.md

```diff
@@ -122,13 +122,13 @@ stub = Helloworld::Greeter::Stub.new('localhost:50051', creds: creds)
 
 ```csharp
 // Base case - No encryption/authentication
-var channel = new Channel("localhost:50051", Credentials.Insecure);
+var channel = new Channel("localhost:50051", ChannelCredentials.Insecure);
 var client = new Greeter.GreeterClient(channel);
 ...
 
 // With server authentication SSL/TLS
-var credentials = new SslCredentials(File.ReadAllText("ca.pem"));  // Load a CA file
-var channel = new Channel("localhost:50051", credentials);
+var channelCredentials = new SslCredentials(File.ReadAllText("ca.pem"));  // Load a CA file
+var channel = new Channel("localhost:50051", channelCredentials);
```

Comment:
I think that you will need an ssl override in this case. Typically, you won't have localhost in your server cert...

The other thing that you could do is show how you would connect to a server with a publicly trusted server (in which case you would not need ca.pem (which would be better named as roots.pem BTW)).